WordPress/gutenberg
# Clean up top toolbar to use same metrics as block toolbar

## What?
Not huge changes, but adjusts how the top toolbar applies borders so that they're more in line with the block toolbar's metrics. In doing so the amount of space between controls is much more consistent across the toolbar, resulting in a bit more space before the horizontal scroll is engaged. Also tidies up the parent selector and the movers so that they are centered vertically. 

Noticed while working on https://github.com/WordPress/gutenberg/pull/61118. 

## Testing Instructions

1. Open at the site editor.
2. Set the top toolbar view from the editor header more menu. 
3. Select a block. 
4. See changes. 

## Screenshots or screencast 

### Before
![CleanShot 2024-04-25 at 19 13 15](https://github.com/WordPress/gutenberg/assets/1813435/9ff15d76-a062-4088-96f8-c14f7b4437e2)

### After
![CleanShot 2024-04-25 at 19 11 53](https://github.com/WordPress/gutenberg/assets/1813435/e6592006-b402-44be-bd60-3e427577e80b)
